packages/core/src/routes/profile/index.ts

```diff
@@ -25,6 +27,19 @@ export default function profileRoutes<T extends UserRouter>(
     return;
   }
 
+  router.get(
+    '/profile',
+    koaGuard({
+      response: userProfileResponseGuard.partial(),
+      status: [200],
```

Comment:
```suggestion
      status: [200, 404],
```

Reply:
This endpoint won't throw 404, because the only query is "find user", and this is guarded by `koaOidcAuth`.